app/routes/admin/delete/dogs-route-helper.js

```diff
@@ -0,0 +1,32 @@
+const { getDogsForDeletion } = require('../../../session/admin/delete-dogs')
+
+const getCombinedSelectedList = (request) => {
+  return getDogsForDeletion(request, 1).concat(getDogsForDeletion(request, 2))
+}
+
+const getDateOverrideQueryString = request => {
+  return request.query.today ? `?today=${request.query.today}` : ''
+}
+
+const getCheckboxSortQueryString = request => {
+  const paramStringPrefix = request.query.today ? '&' : '?'
+  const toggledSortOrder = request.query.sortOrder === 'ASC' ? 'DESC' : 'ASC'
+  const sortOrder = request.query.sortKey === 'selected' ? toggledSortOrder : 'ASC'
+  return `${paramStringPrefix}sortKey=selected&sortOrder=${sortOrder}`
+}
```

Comment:
think URLSearchParams would be better for this - https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams